packages/shared/createInjectionState/index.md

```diff
@@ -131,3 +131,29 @@ const [useProvideCounterStore, useCounterStore] = createInjectionState((initialV
   return { count, double, increment }
 }, { injectionKey: CounterStoreKey })
 ```
+
+## Provide a custom default value
+
+```ts
+// useCounterStore.ts
+import { computed, ref } from 'vue'
+import { createInjectionState } from '@vueuse/core'
+
+// custom defaultValue
+const CounterStoreDefaultValue = undefined
+
+const [useProvideCounterStore, useCounterStore] = createInjectionState((initialValue: number) => {
+  // state
+  const count = ref(initialValue)
+
+  // getters
+  const double = computed(() => count.value * 2)
+
+  // actions
+  function increment() {
+    count.value++
+  }
+
+  return { count, double, increment }
+}, { defaultValue: CounterStoreDefaultValue })
```

Comment:
Can we directly pass `defaultValue` without declaring `CounterStoreDefaultValue`?

Reply:
Yes, of course, are you proposing to change that in the documentation?

Reply:
Yes, I suggest change it.